admin-tools/src/webapp/vm/archive/chef_archive-download.vm

```diff
@@ -6,50 +6,213 @@
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#else
+					<form name="lastpageForm" class="mb-0" method="post" action="#toolForm("$action")">
+						<input type="submit" name="eventSubmit_doList_last" value="&gt;|" disabled="disabled" class="btn btn-secondary btn-sm" />
+						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+					</form>
+				#end
+			#end
+		</div>
+		#end
+	</div>
+
 	#if($archives.size() == 0)
 		<p>$tlang.getString("archive.download.none")</p>
 	#else
-		<table class="table table-hover table-striped table-bordered">
-		<thead>
-			<tr>
-				<th>
-					$tlang.getString("archive.download.site")
-				</th>
-				<th>
-					$tlang.getString("archive.download.date")
-				</th>
-				<th>
-					$tlang.getString("archive.download.size")
-				</th>
-				<th>
-					$tlang.getString("archive.download.hash")
-				</th>
-			</tr>
-		</thead>
-		<tbody>
-		#foreach($archive in $archives)
-			<tr>
-				<td>
-					<a href="/admin-tools/download/?archive=#chef_html($!archive.filename)" class="downloadLink">
-						#if ($archive.siteTitle)
-							#chef_html($!archive.siteTitle) <small>(#chef_html($!archive.siteId))</small>
-						#else 
-							#chef_html($!archive.siteId)
-						#end
-					</a>
-				<td>
-					$!archive.dateCreated
-				</td>
-				<td>
-					$!archive.size
-				</td>
-				<td>
-					$!archive.hash
-				</td>
-			</tr>
+		<div class="table-responsive">
+			<table id="archiveTable" class="table table-hover table-striped table-bordered">
+			<thead>
+				<tr>
+					<th class="sortable" data-sort="text">
+						$tlang.getString("archive.download.site")
+						<i class="bi bi-arrow-down-up"></i>
+					</th>
+					<th class="sortable" data-sort="date">
+						$tlang.getString("archive.download.date")
+						<i class="bi bi-arrow-down-up"></i>
+					</th>
+					<th class="sortable" data-sort="size">
+						$tlang.getString("archive.download.size")
+						<i class="bi bi-arrow-down-up"></i>
+					</th>
+					<th class="sortable" data-sort="text">
+						$tlang.getString("archive.download.hash")
+						<i class="bi bi-arrow-down-up"></i>
+					</th>
+				</tr>
+			</thead>
+			<tbody>
+			#foreach($archive in $archives)
+				<tr>
+					<td>
+						<a href="/admin-tools/download/?archive=#chef_html($!archive.filename)" class="downloadLink">
+							#if ($archive.siteTitle)
+								#chef_html($!archive.siteTitle) <small>(#chef_html($!archive.siteId))</small>
+							#else 
+								#chef_html($!archive.siteId)
+							#end
+						</a>
+					<td>
+						$!archive.dateCreated
+					</td>
+					<td>
+						$!archive.size
+					</td>
+					<td>
+						$!archive.hash
+					</td>
+				</tr>
 
-		#end
-		</tbody>
-	</table>
+			#end
+			</tbody>
+		</table>
+		</div>
 	#end
-</div>
\ No newline at end of file
+</div>
+
+#literal()
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
+    var table = document.getElementById('archiveTable');
```

Comment:
this all looks good. my only comment is that it would be nice if you used let instead of var ..... i've started to associate var with our old legacy code and let tells me that the code has been edited in past couple years!